addons/mail/static/src/models/chat_window_manager/chat_window_manager.js

```diff
@@ -94,6 +94,7 @@ function factory(dependencies) {
                 newMessageChatWindow = this.env.models['mail.chat_window'].create({
                     manager: [['link', this]],
                 });
+                this.update({ newMessageChatWindow: [['link', newMessageChatWindow]]});
```

Comment:
Hello, I'm not sure to understand the fix, this should be done automatically by `_computeNewMessageChatWindow` already. Is it not called correctly?

Reply:
> Hello, I'm not sure to understand the fix, this should be done automatically by `_computeNewMessageChatWindow` already. Is it not called correctly?

Yes! not called because `allOrdered` is missing on `newMessageChatWindow` field dependencies.